app/views/devise/shared/_links.html.erb

```diff
@@ -18,8 +32,66 @@
   <%= link_to "Didn't receive unlock instructions?", new_unlock_path(resource_name) %><br />
 <% end %>
 
-<%- if devise_mapping.omniauthable? %>
-  <%- resource_class.omniauth_providers.each do |provider| %>
-    <%= link_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider), method: :post %><br />
-  <% end %>
-<% end %>
+
+<style>
+
+  /* Devise Button */
```

Comment:
I think this comment is unnecessary

Reply:
I like to separate the css but ok, I delete the comments